docs/native/src/scenarios/operational_flows.rst

```diff
@@ -0,0 +1,81 @@
+
+Add a Place 
+~~~~~~~~~~~
+This describes the first time a user adds your storage provider using the :guilabel:`Add a Place` in |Office iOS|. To correctly authenticate users, we use the following operational flow in order to connect a user's storage provider account with their |Office iOS| account. 
+
+.. figure:: ../images/user_flows.png
+    :alt: Image representation of the authentication flow for adding a place.
+
+ 
+#. When the Office client boots, it calls the Services Manager for a list of available services. The Services manager returns a Service Catalog, which is an alphabetically-sorted list of available providers and places which the user can connect to. 
+#. When the user clicks :guilabel:`Add a Place` in the |Office iOS| backstage, they see your Storage Provider listed as an available place. 
+#. When the users adds the Storage Provider, the Office client sends an unauthenticated call to the bootstrapper URL.
+#. The Bootstrapper URL responds with a 401 challenge as well as the authorization URI and token issue URI. Please note that the Authorization URI must be provided as part of your onboarding information so we can add it as a trusted domain.
+#. The Office client loads the authorization URI in a web view which prompts the user to sign in with the service provider.  Once the user finishes the sign in process, the web view reaches a redirect URI which causes it to close. The redirect URI also provides auth code to the Office client. 
+#. The Office client sends the auth code & token issue URI to the Office Identity service. 
+#. The Office Identity service sends the auth code and the client secret to the Token endpoint.
+#. The Token Endpoint issues an access token and a refresh token (if available) back to the Office Identity Service.
+#. The Office Identity Service calls the Bootstrapper using the tokens to retrieve the user profile information. 
+#. The Office Identity service sends these tokens & the user profile information to the client. 
+#. The client uses the OAuth code to call the bootstrap shortcut to obtain a Root Container URL. If the shortcut is not implemented, it calls the Ecosystem URL.
+#. The Bootstrapper returns the root container or ecosystem URL, as well as the OAuth token info and profile information to the Office client.
+#. The user is now able to see their files under the Storage Provider's Place and open the Storage Provider files directly in their |Office iOS| app. 
+
+
+Browsing and opening files
+--------------------------
+Here is the operational flow for browsing and opening files.   
+
+#. *Get the RootContainerURL:* 
+
+    * Option 1: Call the :ref:`GetRootContainer (bootstrapper)` shortcut and provide all of the required Auth headers. 
+    * Option 2: If the :ref:`GetRootContainer (bootstrapper)` shortcut is not implemented, call :ref:`GetRootContainer (ecosystem)`. If this call doesn't contain ContainerInfo, use the WOPI URL to call :ref:`CheckContainerInfo`. 
```

Comment:
Elsewhere in the docs, it says that the GetRootContainer shortcut operation is required: http://wopi.readthedocs.io/projects/officewopi/en/latest/wopi_requirements.html#bootstrapper